src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -14,4 +14,14 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>EnableAutomaticNaming__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates if naming is automated. When multi-currency is enabled, the name format is: {!DonorName} {ISO Code} {!Amount} - Recurring. Otherwise, {!DonorName} {CurrencySymbol}{!Amount} - Recurring.</description>
```

Comment:
@sam-knox , can you please advise on the "Enable Automatic Naming" Description and Help Text?

Reply:
@sam-knox ^

Reply:
Ok, how about something more simple like:

"If enabled, Salesforce automatically populates the Recurring Donation Name." 

I assume that the name format configuration will eventually be on the same NPSP Settings page as this setting.

Reply:
The Description and Help Text have been updated.